src/main/java/fruitshop/Application.java

```diff
@@ -0,0 +1,51 @@
+package fruitshop;
+
+import fruitshop.db.ReportStorage;
+import fruitshop.db.TransactionStorage;
+import fruitshop.model.Operation;
+import fruitshop.service.file.WriteDataToStorageService;
+import fruitshop.service.impl.FileReadServiceImpl;
+import fruitshop.service.impl.FileWriteServiceImpl;
+import fruitshop.service.impl.GenerateReportDataImpl;
+import fruitshop.service.impl.ProcessDataImpl;
+import fruitshop.service.impl.WriteDataToStorageImpl;
+import fruitshop.strategy.AdditionHandler;
+import fruitshop.strategy.BalanceHandler;
+import fruitshop.strategy.StrategyService;
+import fruitshop.strategy.SubtractionHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Application {
+    private static final WriteDataToStorageService writeDataService = new WriteDataToStorageImpl();
+    private static final FileReadServiceImpl fileReadService
+            = new FileReadServiceImpl(writeDataService);
+    private static final FileWriteServiceImpl fileWriteService = new FileWriteServiceImpl();
+    private static final ProcessDataImpl processData = new ProcessDataImpl();
```

Comment:
```suggestion
    private static final FileWriteService fileWriteService = new FileWriteServiceImpl();
    private static final ProcessData processData = new ProcessDataImpl();
```

let's initialize our services using interfaces, do the same everywhere